Backend/prisma/schema.prisma

```diff
@@ -0,0 +1,69 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
```

Comment:
@SemanBoon Do we plan to store addresses of Users so that we can share relevant hair services from their default location?

Reply:
Yes, I feel like that would make sense but I was thinking of it a different way. I want the search feature to search based on a city or state that the user enters, so when the results pop up and the user clicks on one, it should lead them to the service provider profile. Somewhere on the profile I plan to implement the maps api that should show the address of the service provider and how far it is from the user. I will edit code and add the user address. Thank You!

Reply:
I see! Just from a user experience point of view though, users will find it more convenient if they have a default search area depending on their home address and they still have the option to search for hair services in other locations